Sources/Commands/SwiftPackageTool.swift

```diff
@@ -582,7 +591,7 @@ extension SwiftPackageTool {
                 destination = output
             } else {
                 let graph = try swiftTool.loadPackageGraph()
-                let packageName = graph.rootPackages[0].name
+                let packageName = graph.rootPackages[0].manifestName // TODO: use identity instead?
```

Comment:
Based on the other changes here, it seems that we'll need to use the identity string instead of the name (which isn't necessarily wrong).  That might be a good thing to move to across-the-board.  I wonder if IDEs should also show that in their user interface — there has been this unfortunate ambiguity between the name of the package directory vs the name declared in the manifest.

I had been thinking of the identity as opaque but it probably _is_ the right thing to use in diagnostics and descriptions after all.

Reply:
tried to keep the changes to behavior minimal in this PR -- a follow up PR to clean up the remain usage of `manifestName` and eventually remove `name` from the manifest are next on my list

Reply:
Sounds great!  Thanks for taking this on!